infra/TODO.md

```diff
@@ -4,7 +4,6 @@
 * Better AZN
 * Review VPC strategy
```

Comment:
I'd like to chat about this. Not that I have the answer but i'm interested in the discussion